doc/kubernetes/modules/ROOT/pages/running/keycloak-with-external-infinispan.adoc

```diff
@@ -15,12 +15,12 @@ See xref:running/index.adoc[] for additional guides.
 
 * OpenShift or Kubernetes cluster running.
 * Existing xref:running/keycloak-deployment.adoc[Basic Keycloak deployment] as it will be extended.
-* Existing deployment of Infinispan with the Operator (TODO).
+* Existing Infinispan deployment, for example, one of xref:running/infinispan-deployment.adoc[] or xref:running/keycloak-with-external-infinispan.adoc[].
```

Comment:
Should "infinispan" be "{ispn}"?

Reply:
`ispn` attribute does not exist in this file.